src/qrl/core/p2p/p2pPeerManager.py

```diff
@@ -20,6 +20,14 @@
 
 
 class P2PPeerManager(P2PBaseObserver):
+    """
+    P2PPeerManager is an Observer, and its main role is to bundle together
```

Comment:
Comment for all the functions inside P2PPeerManager that has been registered by Observable.register, will be more helpful.